dags/app_update_out_of_date.py

```diff
@@ -0,0 +1,66 @@
+from airflow import DAG
+from airflow.operators.subdag_operator import SubDagOperator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from datetime import datetime, timedelta
+
+from utils.dataproc import moz_dataproc_pyspark_runner
+
+"""
+
+The following WTMO connections are needed in order for this job to run:
+conn - google_cloud_airflow_dataproc
+conn - aws_dev_iam_s3
```

Comment:
@haroldwoo is this the correct connection for writing to `s3://telemetry-public-analysis-2`? Or we should rather create one specifically for this job?

Reply:
Just checked and that aws user only allows rw to telemetry-parquet and a few other buckets. The bucket you mentioned is not one of them.

Reply:
Can you create a bug and assign to me for the access required by this job? Also can we name this dag something more descriptive than app_update_out_of_date.py?

Reply:
> Can you create a bug and assign to me for the AWS access required by this job?

https://bugzilla.mozilla.org/show_bug.cgi?id=1590189

> Also can we name this dag something more descriptive than app_update_out_of_date.py?

I'll change this to update_orphaning_dashboard_etl, that should be less cryptic.

Reply:
Connection has been updated.
DAG has been renamed.